decidim-core/spec/system/authentication_spec.rb

```diff
@@ -328,6 +328,28 @@
         expect_current_user_to_be(user)
       end
 
+      it "displays error when email is empty and focus shifts to password" do
+        click_on("Log in", match: :first)
+
+        within "#session_new_user" do
+          fill_in :session_user_email, with: ""
```

Comment:
```suggestion
          fill_in :session_user_email, with: ""
          expect(page).not_to have_css(".form-error.is-visible")
```
We should expect not to have the error here before the click right?